File: unpatched/unverifyemail.py
import requests
import sys
token = sys.argv[1]

if not token or token == "":
    raise TypeError(f'python {__file__} <token>')

res = requests.get("https://discord.com/api/guilds/0/members", headers={
    'Authorization': token
})

print(res.text) # "No access"

File: unpatched/unverifyemail.js
const {default:axios} = require("axios")
let token = process.argv[2]
if (!token || token === "") {
    throw console.error(`node ${__filename} <token>`)
}
axios({
    method: "get",
    url: "https://discord.com/api/guilds/0/members",
    headers: {
        authorization: token
    }
}).catch((err) => {
    console.log("Email unverified!")
})
// It will literally send an email like "Thanks for registering an account" when you re-verify your email
